datashuttle/utils/ds_logger.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from pathlib import Path
+from typing import Any, List, Optional
+
+import fancylog as package
+from fancylog import fancylog
+from rich import print as rich_print
+from rich.console import Console
+from rich.filesize import decimal
+from rich.markup import escape
+from rich.text import Text
+from rich.tree import Tree
+
+from . import utils
+
+
+def start(
+    path_to_log: Path, name: str, variables: Optional[List[Any]]
+) -> None:
+    """"""
+    fancylog.start_logging(
+        path_to_log,
+        package,
+        filename=name,
+        variables=variables,
+        verbose=False,
+        timestamp=True,
+        file_log_level="INFO",
+        write_git=False,
+        log_to_console=False,
+    )
+    logging.info(f"Starting {name}")
```

Comment:
I'm not sure this info message would be very clear to the user.

Reply:
thanks have changed to `logging.info(f"Starting logging for command {name}")`